src/router_db.erl

```diff
@@ -46,6 +46,10 @@ start_link(Args) ->
 get() ->
     gen_server:call(?SERVER, get).
 
+-spec get(default | devices | xor_filter_devices) -> {ok, rocksdb:db_handle(), rocksdb:cf_handle()}.
```

Comment:
Maybe do a get_device and get_xor_filter_devices (not sure if we ever use default)

Reply:
I like that idea better :+1: